### DISCUSSION TITLE
New update of canvas when?

### DISCUSSION THREAD

**Discussion by Dande057** [thatskymod/Sky-CotL-Scripts#87] [General]
Pleaseee when will the new update of canvas come??? The event is on monday and i still don't have enough candles 😭😭

**Comment by Xyn-Zurre** [comment]
> Pleaseee when will the new update of canvas come??? The event is on monday and i still don't have enough candles 😭😭

Fr

**Comment by thatskymod** [comment]
### Update finished
All files from the [Libs/](https://github.com/thatskymod/Sky-CotL-Scripts/tree/main/Canvas/Libs) folder have now been replaced by their newer counterparts.

Thank you for standing by while the issues were being addressed.

**Reply by Dande057** [reply]
Thank youuuu,
love you guys ❤️❤️❤️
